### DISCUSSION TITLE
Not able to assign any user in Assignees or Reviewer

### DISCUSSION THREAD

**Discussion by DishaT20** [skills/.github#321] [Review pull requests]
No user profile is visible even after typing correct and complete username in the assignees and reviewer column. If anyone knows the solution  please answer.

**Comment by sinsukehlab** [comment]
Try inviting a user to be a collaborator of your repository.
The docs page for requesting a review could be kinder enough to specify who has read access to a personal repository.

**Comment by anxumalo** [comment]
Hi @DishaT20

> No user profile is visible even after typing correct and complete username in the assignees and reviewer column. If anyone knows the solution please answer.

It looks like you were able to complete the **Reviewing pull requests** exercise since you added yourself as an _assignee_ in this PR: https://github.com/DishaT20/skills-review-pull-requests/pull/1. Nice job! 🎉 

If you were struggling to add a _reviewer_ during the exercise, here are some tips to keep in mind:

- As mentioned in the training, since you created the pull request, you can't assign yourself as a reviewer (I guess you figured that out already! 😄)
  - This works really well if you're part of a team who can review, approve, and merge PRs.
- As @sinsukehlab mentioned, try inviting a user to be a collaborator in your repository.
- Try adding a [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file to your project to automatically assign reviewers to pull requests, and enable required reviews.

Hope this helps! 

ℹ️ _If this answer was satisfactory, feel free to 'mark as answered' in the feed to help other users with similar questions._

**Reply by sinsukehlab** [reply]
This helps a lot. Thank you!

**Reply by anxumalo** [reply]
@sinsukehlab  You're welcome - happy to help! Please mark my previous response as **answered** to help other users with similar questions find the right solution. Thanks in advance 😄
